Sources/SwiftUI-Utils/ScrollView/VisibleViewGetters.swift

```diff
@@ -0,0 +1,76 @@
+import SwiftUI
+
+extension View {
+    public func setLoadedItem(_ itemId: String) -> some View {
+        anchorPreference(
+            key: LoadedItemsPreferenceKey.self,
+            value: .bounds,
+            transform: { anchor in
+                [.init(itemId: itemId, bounds: anchor)]
+            }
+        )
+    }
+}
+
+extension ScrollView {
+    public func readVisibleItems(_ items: @escaping ([String]) -> Void) -> some View {
+        backgroundPreferenceValue(LoadedItemsPreferenceKey.self) { loadedItems in
+            ReadVisibleItemView(loadedItems: loadedItems)
+        }
+        .onPreferenceChange(VisibleItemsPreferenceKey.self) {
+            items($0)
+        }
+    }
+
+    public func readVisibleItems(_ items: Binding<[String]>) -> some View {
+        backgroundPreferenceValue(LoadedItemsPreferenceKey.self) { loadedItems in
+            ReadVisibleItemView(loadedItems: loadedItems)
+        }
+        .onPreferenceChange(VisibleItemsPreferenceKey.self) {
+            items.wrappedValue = $0
+        }
```

Comment:
Tu devrais pouvoir juste réutiliser ta fonction avec une _closure_:


```suggestion
        onVisibleViewIdentifiersChange {
            items.wrappedValue = $0
        }
```